### ISSUE TITLE
About Training Data

### ISSUE THREAD

**Issue by ersinyar** [chen0040/keras-text-summarization#2]
Hello, thank you for sharing your codes. There are pre-trained model files in demo folder. Did you use fake_or_real_news.csv while training these models? Or did you use another input data?

**Comment by chen0040** [comment]
@ersinyar i used fake_or_real_news.csv  while training these models for the scripts in the demo folder

**Comment by ZequnSong** [comment]
@chen0040 Hello, about the training data, I think I found that there are many format errors in  fake_or_real_news.csv, and while using seq2seq_train.py to train, I only got 0.10 accuracy. Is this a normal result? Could you re-upload a training data? look forward to your kind reply ^_^

**Comment by gunju** [comment]
Even I am getting very less accuracy about 0.1735. Even after increasing the epochs, there is no improvement.

**Comment by SampannaKahu** [comment]
Even I am getting very low accuracy with the fake_or_real_news dataset.

**Comment by videetnimsarkar21** [comment]
Please respond if you have solved the problem.

**Comment by ICRL-lab** [comment]
Did any one got improved results using this repo?
